tools/depcheck/pkg/cmd/trace/pkg.go

```diff
@@ -0,0 +1,131 @@
+package trace
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gonum/graph"
+	"github.com/gonum/graph/concrete"
+
+	depgraph "github.com/openshift/origin/tools/depcheck/pkg/graph"
+)
+
+var (
+	// lets us distinguish core library deps
+	knownVendorRoots = []string{
+		"/vendor/bitbucket.org/",
+		"/vendor/cloud.google.com",
+		"/vendor/github.com",
+		"/vendor/go4.org",
+		"/vendor/golang.org",
+		"/vendor/google.golang.org",
+		"/vendor/gopkg.in",
+		"/vendor/k8s.io/",
+		"/vendor/vbom.ml",
+	}
+	baseRepo = "github.com/openshift/origin"
```

Comment:
This needs to be a parameter to whatever function you're using it in.  It's logically an argument to the command